app/screens/bottom_sheet/index.tsx

```diff
@@ -19,24 +19,27 @@ import Indicator from './indicator';
 
 type SlideUpPanelProps = {
     closeButtonId?: string;
+    componentId?: string;
     initialSnapIndex?: number;
     renderContent: () => ReactNode;
     snapPoints?: Array<string | number>;
 }
 
-const BottomSheet = ({closeButtonId, initialSnapIndex = 0, renderContent, snapPoints = ['90%', '50%', 50]}: SlideUpPanelProps) => {
+const BottomSheet = ({closeButtonId, componentId: screenComponentId, initialSnapIndex = 0, renderContent, snapPoints = ['90%', '50%', 50]}: SlideUpPanelProps) => {
     const sheetRef = useRef<RNBottomSheet>(null);
     const dimensions = useWindowDimensions();
     const isTablet = useIsTablet();
     const theme = useTheme();
     const lastSnap = snapPoints.length - 1;
 
+    const componentId = isTablet && screenComponentId ? screenComponentId : Screens.BOTTOM_SHEET;
```

Comment:
in any case this is not needed as `BOTTOM_SHEET` would already be the value of the `componentId`.